recipes-graphics/mesa/mesa-etnaviv-env_0.1.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "Mesa environment variables for etnaviv on xserver"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit allarch
```

Comment:
I think we ought to change this to use `MACHINE_ARCH` and to provide a default environment for use with `etnaviv` for the `use-mainline-bsp` override so we can include the package as `RRECOMMENDS` of one of `mesa` core libraries so it would be included for all images which had a graphical component but wouldn't hurt machines using other SoC.

Reply:
* Why MACHINE_ARCH? What gets better then? We are talking about a plain config file.
* Will add libdrm-etnaviv + mesa-etnaviv-env to RDEPENS of a mesa core library

Reply:
The file need to be different depending if it is an i.MX6 or a TI device, for example.

Reply:
mesa-etnaviv-env_0.1.bb on TI device??? This recipe is for one specific use case and it is not a generic mesa-env. Your suggestion buys us that non-machine-specific mesa RRECOMENDS machine-specific mesa-etnaviv-env. Yes we can add magic SIGGEN_EXCLUDE.. but doing this should be an exception.

Reply:
I know it is. But we need to ensure this wouldn't contaminate other machines. We just need to verify it.